common/check_free_space

```diff
@@ -9,28 +9,29 @@
 # - Vincent Garonne, <vincent.garonne@cern.ch>, 2015
 
 '''
-Probe to check the min free space.
+Probe to check the free space.
 '''
 
 import sys
 import traceback
-
-from rucio.core.rse import list_rses, get_rse_limits, set_rse_usage
+from rucio.core import monitor
+from rucio.core.rse import list_rses, get_rse_usage
 
 # Exit statuses
 OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
 
 
 if __name__ == "__main__":
     try:
-
         for rse in list_rses():
-            limits = get_rse_limits(rse=rse['rse'], rse_id=rse['id'])
-            min_free_space = limits.get('MinFreeSpace')
-            if min_free_space is not None:
-                # print rse['rse'], min_free_space
-                set_rse_usage(rse=rse['rse'], source='min_free_space', used=min_free_space, free=None)
+            limits = get_rse_usage(rse['id'])
+            for dict in limits:
```

Comment:
dict is a reserved word in python and should not be used as a variable. Also, generally, you don't name variables in python after the types they are. So maybe just call it 'usage'?